src/main/java/frc/robot/subsystems/LimeVision/LimeLightSub.java

```diff
@@ -52,9 +85,22 @@ public class LimeLightSub extends SubsystemBase {
   public LimeLightSub(String limelight_networktable_name) {
     limelight_comm = new LimeLightComms(limelight_networktable_name);
     limelight_comm.set_entry_double("ledMode", 3);
+    Shuffleboard.getTab("Limelight").addDouble("BotPose TX", ()->getBlueBotPose()[0]);
```

Comment:
Can we remove the Shuffleboard stuff if it is just used for testing?

Reply:
Adding a 'verbose' flag